src/include/daos_srv/container.h

```diff
@@ -71,7 +71,7 @@ struct ds_cont_child {
 	    sc_dtx_delay_reset : 1, sc_dtx_registered : 1, sc_props_fetched : 1, sc_stopping : 1,
 	    sc_destroying : 1, sc_vos_agg_active : 1, sc_ec_agg_active : 1,
 	    /* flag of CONT_CAPA_READ_DATA/_WRITE_DATA disabled */
-	    sc_rw_disabled : 1, sc_scrubbing : 1, sc_rebuilding : 1, sc_open_initializing : 1;
+	    sc_rw_disabled : 1, sc_scrubbing : 1, sc_rebuilding : 1;
```

Comment:
mentioned on your last patch but if you change these to semicolons, clang-format will look better and it's the same behavior.

Reply:
ah, sure, missed that. Thanks!